overlord/hookstate/ctlcmd/get.go

```diff
@@ -55,13 +55,12 @@ The get command prints configuration options for the current snap.
     $ snapctl get username
     frank
 
-If multiple option names are provided, a document is returned:
+If multiple option names are provided, the corresponding values are returned:
 
     $ snapctl get username password
-    {
-        "username": "frank",
-        "password": "..."
-    }
+    Key       Value
+    username  frank
```

Comment:
That's a wrong location. AFAICT the fix should be in `cmd/snap/cmd_get.go` as https://bugs.launchpad.net/ubuntu/+source/snapd/+bug/1776466 is about `snap get`.  `snapctl` does return a JSON document.

Reply:
Oops. will do it over.